ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dynamic-prefix-lists.j2

```diff
@@ -0,0 +1,12 @@
+{# eos - dynamic prefix-lists #}
+{% for dynamic_prefix_list in dynamic_prefix_lists | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
```

Comment:
Is this a new feature or do we have an old one? If this is a new one then instead of using ```arista.avd.convert_dicts('name') ``` use the new data-model itself.

Will suggest the new data-model in README.